restful-web-services/src/main/java/com/thinkitive/restfulwebservices/filtering/FilteringController.java

```diff
@@ -0,0 +1,37 @@
+package com.thinkitive.restfulwebservices.filtering;
+
+import com.fasterxml.jackson.databind.ser.FilterProvider;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
+import org.springframework.http.converter.json.MappingJacksonValue;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class FilteringController {
+    @GetMapping("/filtering")
+    public MappingJacksonValue filtering(){
```

Comment:
We can take filtering parameter from QueryParam instead hard-coded.
For now it only filtering field1 and field3 parameter